src/components/bookList.js

```diff
@@ -29,27 +29,33 @@ const BookList = () => {
     <div className="imp-book">
       {book.map((key) => (
         <div key={key.item_id} className="book-cont">
-          <div>
-            <h4>{key.title}</h4>
-            <p>{key.author}</p>
-            <button type="button">comments</button>
-            <button type="button" onClick={() => handleremoveBook(key.item_id)}>remove</button>
-            <button type="button">edit</button>
+          <div className="author-section">
+            <div className="book-author">
+              <p className="cat">{key.category}</p>
+              <h4 className="aut">{key.title}</h4>
+              <p className="tit">{key.author}</p>
+            </div>
+            <div className="book-btn">
+              <button type="button" className="l-btn">comments</button>
+              <button className="l-btn rem" type="button" onClick={() => handleremoveBook(key.item_id)}>remove</button>
+              <button type="button" className="l-btn">edit</button>
+            </div>
           </div>
-          <div>
-            <AiOutlineLoading3Quarters />
-            <div>
-              <span>64%</span>
-              <span>completed</span>
+          <div className="comp-icon">
+            <div className="progress">
+              <AiOutlineLoading3Quarters />
+            </div>
+            <div className="prog-details">
+              <span className="comp-percent">64%</span>
+              <span className="comp-status">completed</span>
             </div>
           </div>
-          <div>
-            <p>CURRENT CHAPTER</p>
-            <p>
-              Chapter
-              {book.chapter}
+          <div className="current-cont">
+            <p className="current-chpt">CURRENT CHAPTER</p>
+            <p className="chapter-1">
+              Chapter 15
             </p>
-            <button type="button">UPDATE PROGRESS</button>
+            <button type="button" className="upBtn">UPDATE PROGRESS</button>
           </div>
```

Comment:
- [ ] Good job so far on styling the books. There are a few styles and placeholders missing. Kindly align the items in the right to left alignment and not center alignment in the Current Chapter section so your design match with the original design. 👍🏽
- [ ] There is a border separator between the percentage and the current chapter section. Kindly add that 👍🏽

**Please refer to the screenshot below for the section I asked for changes above** 

**Your Design**

<img width="1711" alt="Screenshot 2023-05-27 at 9 52 14 PM" src="https://github.com/emmanuelsteven/BookStore-React-Redux/assets/50941074/47258206-2270-4864-b47b-2010cae1f869">

**The Original Design**

<img width="1094" alt="Screenshot 2023-05-27 at 9 59 40 PM" src="https://github.com/emmanuelsteven/BookStore-React-Redux/assets/50941074/a8fc5a7d-cf4a-41b7-97ff-000a4be48d5a">